index.html

```diff
@@ -118,19 +204,55 @@
     // TRANSFORM
     txData = data.slice();
 
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
+    if (options.party !== 'all') {
+      var party = options.party;
       txData = txData.filter(function (d) {
-        return d.year >= yearRange[0] && d.year <= yearRange[1];
+        return d.party === party;
       });
     }
 
     // SCATTERPLOT
+    var points = svg.selectAll('.point')
+      .data(txData);
+
+    points.enter().append('circle')
+      .classed("point", true)
+      .classed("R", function (d){
+        if (d.party === "rep") {return true}
```

Comment:
So we're returning `true` if `d.party === "rep"`. But `d.party === "rep"` itself gives us a true/false value. Could we simplify this to obviate the need for an if statement?